go/inst/instance_dao.go

```diff
@@ -242,18 +242,17 @@ func (instance *Instance) checkMaxScale(db *sql.DB, latency *stopwatch.NamedStop
 }
 
 // areReplicationThreadsRunning checks if both IO and SQL threads are running
-func areReplicationThreadsRunning(instanceKey *InstanceKey) (replicationThreadsRunning bool, err error) {
+func areReplicationThreadsRunning(instanceKey *InstanceKey) (ioThreadRunning, sqlThreadRunning bool, err error) {
```

Comment:
Not sure on go syntax, but would this be more right?
```suggestion
func areReplicationThreadsRunning(instanceKey *InstanceKey) (ioThreadRunning bool, sqlThreadRunning bool, err error) {
```

Reply:
Both are correct, and I like yours better.